sympy/functions/special/spherical_harmonics.py

```diff
@@ -237,6 +237,29 @@ def Ynm_c(n, m, theta, phi):
     .. math::
         \overline{Y_n^m(\theta, \varphi)} := (-1)^m Y_n^{-m}(\theta, \varphi).
 
+    Examples
+    ========
+
+    >>> from sympy import Ynm_c, Symbol, simplify
+    >>> from sympy.abc import n,m
+    >>> theta = Symbol("theta")
+    >>> phi = Symbol("phi")
+
```

Comment:
I'm not sure how good an idea these spaces are. 
Although, the examples look better in source, wouldn't it render as different shells?

Can you post some pictures of how the examples are rendered?
You can check https://docs.sympy.org/dev/documentation-style-guide.html#build-the-documentation for building the documentation locally.

Reply:
I am building the docs and checking locally everytime before adding commits to this PR.

Reply:
![Screenshot from 2020-06-03 12-52-51](https://user-images.githubusercontent.com/51290447/83607736-5dc51d80-a599-11ea-8c86-1324d7e37aae.png)

Reply:
This is how it is rendering each example

Reply:
Yeah, the examples are rendering as different shells. 
I don't think the tiny bit of source readability is worth rendering the examples as different shells.